src/components/projects.vue

```diff
@@ -146,6 +199,27 @@ export default defineComponent({
       }
     };
 
+    const sortAlphabetically = (a: { name: string }, b: { name: string }) => {
+      return a.name.localeCompare(b.name);
+    };
+
+    const sortOrderProjects = ref<SortOrder>(SortOrder.ASC);
+    const sortOrderProjectGroups = ref<SortOrder>(SortOrder.ASC);
```

Comment:
move this at the top in the variable definitions (after fetchAbort) and use the standard way to define them:

```
    const sortOrderProjects: Ref<SortOrder> = ref(SortOrder.ASC);
    const sortOrderProjectGroups: Ref<SortOrder> = ref(SortOrder.ASC);
```